### DISCUSSION TITLE
Opensearch Pass through logon details

### DISCUSSION THREAD

**Discussion by rhysxevans** [grafana/grafana#68040] [Data Sources]
Hi

I have a setup where opensearch (via cross cluster search) has a number of roles, and the users are authenticated via Azure AD (oauth). This all works when using kibana and users are able to view their restricted data. 

Is there a way to make the opensearch plugin do the same. IE it only shows the data the person should see. ?

I have tried a number of things (short of putting auth proxies between grafana and opensearch).

Is there anyone who has been able to make this setup work? Any help is greatly appreciated 

Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
